cms/topmenu.json

```diff
@@ -1,8 +1,7 @@
 {
   "items": [
-    {"Home": "http://apps.loklak.org"},
+  	{"About": "http://loklak.net/about"},
```

Comment:
yes, i have intended - "convert to spaces" on whole, where ever i made changes! But, still its failing because there's lot more files need to be fixed